block/manager.go

```diff
@@ -243,3 +253,49 @@ func (m *Manager) UpdateTargetHeight(h uint64) {
 		}
 	}
 }
+
+// ValidateConfigWithRollappParams checks the configuration params are consistent with the params in the dymint state (e.g. DA and version)
+func (m *Manager) ValidateConfigWithRollappParams() error {
+	if version.Commit != m.State.ConsensusParams.Params.Commit {
+		return fmt.Errorf("binary version mismatch. rollapp param: %s binary used:%s", version.Commit, m.State.ConsensusParams.Params.Commit)
+	}
+
+	if m.DAClient == nil {
```

Comment:
why is this validated here? we're already validating this in the setDa.

Reply:
i think if forgot to remove it after a refactor